server/cluster/cluster.go

```diff
@@ -444,6 +446,32 @@ func (c *RaftCluster) RemoveSuspectRegion(id uint64) {
 	c.suspectRegions.Remove(id)
 }
 
+// AddSuspectKeyRanges add the key ranges with the its ruleID as the key
+// The distribution for the keyRanges is like following:
+// [4][]byte: new version start key/new version end key/old version start key/old version end key
+func (c *RaftCluster) AddSuspectKeyRanges(storeID string, keyRanges [4][]byte) {
```

Comment:
can `[][2][]byte` instead of `[4][]byte` be better?

Reply:
good idea